lib/matplotlib/tests/test_axes.py

```diff
@@ -1506,6 +1507,38 @@ def test_bar_tick_label_multiple_old_alignment():
            align='center')
 
 
+@check_figures_equal(extensions=["png"])
+def test_bar_decimal_center(fig_test, fig_ref):
+    # Test vertical, align-center bar chart with Decimal() input
+    # No exception should be raised
+    ax = fig_test.subplots()
```

Comment:
Just for symmetry, please move this down to `# Test image`-

Reply:
Thank you @timhoffm for advice. Have moved it down.